### PR TITLE
Update proxy-setup.adoc

### PR DESCRIPTION
**Pull request by mantel**

### CODE REVIEW

**Comment by keichwa** [comment]
Thanks for spotting, Hubert.  Does it look better this way?

I also tried to fix it in the 3.2 branch.

File: modules/installation/pages/proxy-setup.adoc [hunk 1]
```diff
@@ -94,18 +94,21 @@ The default value here is the proxy's hostname.
...
+State::
+Further information attached to the proxy's certificate.
+
+Country Code::
 In the [guimenu]``country code`` field, enter the country code set during the {productname} installation.
 For example, if your proxy is in the US and your {productname} is in DE, enter `DE` for the proxy.
-====
 +
 
+[TIP] .Country Code
```
Comment (Loquacity) [inline modules/installation/pages/proxy-setup.adoc]:
```suggestion
[NOTE]
```

File: modules/installation/pages/proxy-setup.adoc [hunk 2]
```diff
@@ -94,18 +94,21 @@ The default value here is the proxy's hostname.
...
+====
+Beware the country code must be made of two upper case letters.
```
Comment (Loquacity) [inline modules/installation/pages/proxy-setup.adoc]:
```suggestion
The country code must be two upper case letters.
```

File: modules/installation/pages/proxy-setup.adoc [hunk 3]
```diff
@@ -94,18 +94,21 @@ The default value here is the proxy's hostname.
...
+For further information on country codes, refer to the online https://www.iso.org/obp/ui/#search[list of alpha-2 codes].
```
Comment (Loquacity) [inline modules/installation/pages/proxy-setup.adoc]:
```suggestion
For a complete list of country codes, see https://www.iso.org/obp/ui/#search.
```

**Pull request merged**